webapp/src/controllers/account.ts

```diff
@@ -97,6 +98,14 @@ export async function login(req, res) {
 		if (passwordMatch === true) {
 			const token = await jwt.sign({ accountId: account._id }, process.env.JWT_SECRET); //jwt
 			req.session.token = token; //jwt (cookie)
+
+			if (account.onboarded === false) {
```

Comment:
Seems fine as long as we only want people to hit onboarding after login. 

Because skipping is not required to bypass onboarding, if somebody removed /onboarding from the URL (for any reason), they could be using the app for days or weeks until they login again and are suddenly presented with onboarding again. Our sessions `resave`, they will be valid for 7 days from the time of last use, not login i.e a session can last forever.

To be more strict, you could add a middleware similar to `checkSession` on any page rendering routes.

Reply:
added checkonboardedstatus middleware to check if user isnt onboarded and redirect user to /onboarding page